account_reconcile_payment_order/tests/test_account_reconcile_payment_order.py

```diff
@@ -31,17 +25,29 @@ def setUpClass(cls):
         cls.invoice2.payment_mode_id = cls.inbound_mode
         cls.invoice2.action_post()
         # Add to payment order using the wizard
-        cls.env["account.invoice.payment.line.multi"].with_context(
-            active_model="account.move", active_ids=cls.invoice2.ids,
-        ).create({}).run()
+        res = (
+            cls.env["account.invoice.payment.line.multi"]
+            .with_context(active_model="account.move", active_ids=cls.invoice2.ids)
+            .create({})
+            .run()
+        )
+        payment_order = cls.env["account.payment.order"].browse(res["res_id"])
         # Prepare statement
         cls.statement = cls.env["account.bank.statement"].create(
             {
                 "name": "Test statement",
                 "date": "2019-01-01",
                 "journal_id": cls.bank_journal.id,
                 "line_ids": [
-                    (0, 0, {"date": "2019-01-01", "name": "Test line", "amount": 230}),
+                    (
+                        0,
+                        0,
+                        {
+                            "date": "2019-01-01",
+                            "name": "Test line",
+                            "amount": payment_order.total_company_currency,
```

Comment:
This goes against the nature of the tests. We should fix data.

Reply:
Ok, I think it should be good now, along with https://github.com/OCA/bank-payment/pull/797 (this tests use the base from that module)

Reply:
PR it's complete now, it's ready to merge?

Reply:
It should be completed with the latest change, although test here will only pass after https://github.com/OCA/bank-payment/pull/798 is merged